scripts/setup-openbao-edge-functions.sh

```
@@ -0,0 +1,249 @@
+
+if [[ $DRY_RUN -eq 0 ]]; then
+  $CLI status >/dev/null 2>&1 || {
+    echo "ERROR: $CLI status failed — set BAO_ADDR/BAO_TOKEN (or VAULT_*) and log in" >&2
+    exit 1
+  }
+fi
+
+# Source the env file; only the variables we look up cross back.
+# shellcheck disable=SC1090
+set -a
+. "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)